code-challenges.test.js

```diff
@@ -88,7 +229,18 @@ var vowelTester3 = "challenge"
 // Expected output: 2
 
 
-
-
-
 // b) Create the function that makes the test pass.
+
+//create a function called firstVowel
+//takes in an argument that is a string
+const firstVowel = (str) => {
+    //the string will need to be split in order to give the function the ability to iterate through the indexes
+    var makeStrArr = str.split("");
+    //using the map method I iterate through strArr an have it return a boolean for each value
+    let findVowelIndex = makeStrArr.map(value =>{
+        //this will make an array of true or false
+        return value === "a" || value === "e" || value === "i" || value === "o" || value === "u"
```

Comment:
gonna throw you on a bone on this one since you mentioned not quite being able to used an array
```suggestion
        return ["a", "e", "i", "o", "u"].includes(value)
```
Here's `.includes()` takes in any value as an argument and checks if that provided value matches any items in the array, then returns true if there is a match, false if there's no match